doc: clarify how 26.05's wpa_supplicant changes affect the ctrl_interface (#529993)
NixOS/nixpkgs: nixos/doc/manual/release-notes/rl-2605.section.md

  If client certificates, keys or other files are needed, these should be stored under `/etc/wpa_supplicant` and owned by `wpa_supplicant` to ensure the daemon can read them.

>   Similarly, the `ctrl_interface` directory set in `wpa_supplicant`'s conf must be writeable by the `wpa_supplicant` user so that the `wpa_supplicant` daemon can start successfully. Remove overwrites to the `ctrl_interface` set in an [extraConfig](#opt-networking.wireless.extraConfig) or in a `/etc/wpa_supplicant/imperative.conf`.

Reviewer: ```suggestion
  Similarly, the `ctrl_interface` directory set in `wpa_supplicant`'s conf must be writeable by the `wpa_supplicant` user so that the `wpa_supplicant` daemon can start successfully. If you were changing `ctrl_interface` in [extraConfig](#opt-networking.wireless.extraConfig) or in `/etc/wpa_supplicant/imperative.conf`, please remove that line.
```
